crates/ruff/src/rules/pyupgrade/rules/redundant_open_modes.rs

```diff
@@ -119,8 +119,12 @@ fn match_open<'a>(
 ) -> Option<(Option<&'a Expr>, &'a [Keyword])> {
     let ast::ExprCall {
         func,
-        args,
-        keywords,
+        arguments:
+            Arguments {
+                args,
+                keywords,
+                range: _,
+            },
```

Comment:
I tended to favor destructuring to minimize code changes (i.e., we didn't have to change this method at all, only the `let` match here).